doc/smart-switch/high-availability/smart-switch-ha-hld.md

```diff
@@ -0,0 +1,2208 @@
+# SmartSwitch High Availability High Level Design
+
+| Rev | Date | Author | Change Description |
+| --- | ---- | ------ | ------------------ |
+| 0.1 | 08/09/2023 | Riff Jiang | Initial version |
+| 0.2 | 08/10/2023 | Riff Jiang | Simpify ENI-level traffic control, primary election algorithm |
+| 0.3 | 08/14/2023 | Riff Jiang | Adding DPU level standalone support |
+| 0.4 | 08/17/2023 | Riff Jiang | Redesign HA control plane data channel |
+
+1. [1. Background](#1-background)
```

Comment:
The ToC is a little odd, it has numbered values preceding the heading numbers. Have you tried using the "Markdown All-in-one" extension for VScode? It doesn't assign the heading numbers but it does format the ToC nicely.

Reply:
yep, i am using VSCode with Markdown All-in-one. And the heading numbers are automatically added and updated by it.

![image](https://github.com/sonic-net/SONiC/assets/1533278/e6f68a94-6590-4b7c-9a80-045c7770f7e5)